src/Response.php

```diff
@@ -24,6 +24,13 @@ public function __construct($component, $props, $rootView = 'app', $version = nu
         $this->version = $version;
     }
 
+    public function withStatusCode($statusCode)
```

Comment:
Laravel has a `setStatusCode` method on the response factory, maybe we should use the same method name?

Reply:
Yeah, why not?

Reply:
Done